src/Microsoft.FeatureManagement/FeatureManager.cs

```diff
@@ -44,46 +43,127 @@ private class ConfigurationCacheItem
             public object Settings { get; set; }
         }
 
+        /// <summary>
+        /// Create a feature manager
+        /// </summary>
+        /// <param name="featureDefinitionProvider">The provider of feature flag definitions.</param>
+        /// <param name="options">Options controlling the behavior of the feature manager.</param>
```

Comment:
A throws statement would be valuable to detail that IFeatureDefinitionProvider and the options are required.